src/App/Components/Notifications/Notifications.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/forbid-prop-types */
```

Comment:
you are disabling many rules

Reply:
I thought this one could work but it actually didn't
removing it

Reply:
@nkpremices what issues are you having with the rules?

Reply:
Hound was actually complaining about some proptypes that I was using. But I've managed to solve those issues. I'm going to push so that you can see my new changes @MemunaHaruna